src/tools/ScenarioMeasurement/Startup/Perfcollect.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.Diagnostics.Tracing.Parsers;
+using ScenarioMeasurement;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace ScenarioMeasurement
+{
+    public class Perfcollect : IDisposable
+    {
+        private readonly string filepath = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).FullName, @"Startup/perfcollect");
```

Comment:
nit: instead of ```filepath```, which should be ```filePath```, maybe ```perfCollectScript```

Reply:
Great.